### ISSUE TITLE
Navbar should be positioned fixed for small devices

### ISSUE THREAD

**Issue by jai-k-gohil** [covid19india/covid19india.github.io#1600]
**Navbar Enhancement. Minor fix**
Navbar should be positioned as fixed for small mobile devices for accessibility and better ui/ux.

**Solution**
Check If Screen width is less than 769px, if yes then set navbar to be positioned as fixed.

**Comment by jeremyphilemon** [comment]
Addressed by #1477
